cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -505,10 +505,7 @@ describe('Dashboards', () => {
     cy.get('@org').then(({id}: Organization) => {
       cy.fixture('routes').then(({orgs, dashboards}) => {
         cy.visit(`${orgs}/${id}${dashboards}/${nonexistentID}`)
-        cy.url().should(
-          'eq',
-          `${Cypress.config().baseUrl}${orgs}/${id}/dashboards-list`
-        )
+        cy.url().should('include', `${orgs}/${id}/dashboards-list`)
```

Comment:
why is this one different than the checks one above?

Reply:
two different ways of doing the same thing, which one would you prefer?

Reply:
no preference, just consistency

Reply:
this is exactly the kind of code I should not be writing in the UI repo. Feel free to suggest a preference.

Reply:
matched checks code to this code.